api/types/auth.go

```diff
@@ -25,5 +33,5 @@ type AuthNResponse struct {
 	IdentityToken string `json:"identity_token" example:"a332139d39b89a241400013700e665a3"`
 	// The status of the authentication
 	// Required: true
-	Status string `json:"status" example:"Login successfully"`
+	Status AuthStatus `json:"status" example:"Login successfully"`
```

Comment:
Then you should update the code in envd too.

Reply:
fixed